broker_interface/start_descriptor.py

```diff
@@ -0,0 +1,34 @@
+"""
+Start the consumer process for the file descriptions
+"""
+import os
+import sys
+
+from broker_interface.file_description_consumer import FileDescriptionConsumer
+
+# The Kafka instance host and port are loaded from the environment
+# variables
+KAFKA_HOST = os.environ.get('KAFKA_HOST', 'localhost')
+KAFKA_PORT = os.environ.get('KAFKA_PORT', '9092')
+
+def check_file_path(path):
+    """Makes sure that folders in path exist."""
+    try:
+        os.makedirs(path)
+    except FileExistsError:
+        pass
```

Comment:
O que acha de usar esse parâmetro? Acredito que fica menos poluído
```suggestion
        os.makedirs(path, exist_ok=True)
```

Reply:
Update: Observei que essa função já está presente no crawling utils: https://github.com/MPMG-DCC-UFMG/C01/blob/issue-437/src/crawling_utils/crawling_utils/crawling_utils.py#L52-L57

O que acha de reaproveitá-la? Existe alguma limitação?

Reply:
O módulo `crawling_utils` não é incluído no container do FileDescriptor, por isso não usei essa função. Acredito que incluir esse módulo pode levar a um certo overhead de espaço e tempo de build, mas nada exagerado, e acho que não é problema já que esses recursos não são críticos nesse caso. Porém, como vamos usar uma única função do módulo, ainda não sei se valeria a pena. Pensei em usar a primeira solução que você propôs, com `os.makedirs(path, exist_ok=True)`, o que acha?